webpack/download.hbs

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="defaultLanguage" content="en-US">
+  <meta name="availableLanguages" content="{{availableLanguages}}">
+
+  <meta property="og:title" content="{{title}}"/>
+  <meta name="twitter:title" content="{{title}}"/>
+  <meta name="description" content="{{description}}"/>
+  <meta property="og:description" content="{{description}}"/>
+  <meta name="twitter:description" content="{{description}}"/>
+  <meta name="twitter:card" content="summary"/>
+  <meta property="og:image" content="{{baseUrl}}${require('../public/resources/send-fb.jpg')}"/>
+  <meta name="twitter:image" content="{{baseUrl}}${require('../public/resources/send-twitter.jpg')}"/>
+  <meta property="og:url" content="{{baseUrl}}"/>
+
+  <title>{{title}}</title>
+
+  <link rel="stylesheet" type="text/css" href="${require('../frontend/src/main.css')}" />
+
+  <link rel="icon" type="image/png" href="${require('../public/resources/favicon-32x32.png')}" sizes="32x32" />
+  <link rel="localization" href="/locales/{locale}/send.ftl">
+
+  <script src="/jsconfig.js"></script>
+  <script src="/polyfill.min.js"></script>
+  <script defer src="/l20n.min.js"></script>
```

Comment:
Do these 3 `<script>` tags need to be in the `<head>`, or can we move them to the end of the `<body>` tag? In my very unscientific testing, this had a decent sized performance improvement when testing locally using Lighthouse, but that isn't always a great measure.